test/Unit/PowerShell/PowerShellManagerTests.cs

```diff
@@ -149,6 +149,14 @@ public void ModulePathShouldBeSetCorrectly()
             string workerModulePath = Path.Join(AppDomain.CurrentDomain.BaseDirectory, "Modules");
             string funcAppModulePath = Path.Join(FunctionLoader.FunctionAppRootPath, "Modules");
             string expectedPath = $"{funcAppModulePath}{Path.PathSeparator}{workerModulePath}";
+
+            if (!string.IsNullOrWhiteSpace(Environment.GetEnvironmentVariable("WEBSITE_INSTANCE_ID"))
```

Comment:
Since nothing in the tests set this environment variable, I think we don't need to add this `if` at all here. 

Consider removing this `if` statement or write a new test that will set the environment variable and test the path is in the `PSModulePath`.

Reply:
Removed this condition from the test.